telemetry/specs/INT.mdk

```diff
@@ -652,8 +652,32 @@ both INT header stacks carry information for respective layers and need not be
 considered interfering with each other.
 
 A field in Ethernet, IP, or TCP/UDP should indicate if the
-INT header exists after the TCP/UDP header. We propose two options.
-
+INT header exists after the TCP/UDP header. We propose three options.
+
+* UDP destination port field: a new UDP port number (INT_TBD) will be assigned
+by IANA to indicate the existence of INT after UDP. This option supports two
+use cases:
```

Comment:
In both cases, traffic with INT headers is likely to be hashed to a different path in the network in ECMP scenarios.

Reply:
Text was added below.